Wash-Ups.md

```diff
@@ -0,0 +1,13 @@
+A wash-up can be held after any significant event that has happened where we believe actions can be taken so that we can better improve process moving forward.  This could be following an expedite issue or at the end of a large project.
+
+A new wash-up meeting template has been created that should help people to prepare for the meeting and ensure that all information on why the meeting is taking place has been filled in and circulated to all relevant parties in advance.
+
+https://docs.google.com/document/d/1iYaqVGujJ8ARBgKtFsYxXgswjkdnwxB4flneT6m_9uM/edit
+
+The wash-up meeting itself should be totally focused on what we have learned and what we can improve for next time.
+
+For extra guidance on running a wash-up, please check the Expedite website here: https://sites.google.com/holidayextras.com/hxexpediteprocedure/how-do-i-run-a-wash-up
```

Comment:
same as above, this could be rewritten as:
```
For extra guidance on running a wash-up, please check the [Expedite website](https://sites.google.com/holidayextras.com/hxexpediteprocedure/how-do-i-run-a-wash-up)
```